airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -139,13 +143,12 @@ def execute(self, context: Context):
             self.log.info("Cluster %r in state: %r.", self.cluster_name, cluster_state)
         elif self.deferrable:
             self.defer(
-                trigger=ClusterWaiterTrigger(
-                    waiter_name="cluster_active",
+                trigger=ClusterActiveTrigger(
                     cluster_arn=cluster_details["clusterArn"],
```

Comment:
I'll make the same suggestion I made in Syed's PR... feel free to ignore it, but I see a lot of repetition in the calls to the Triggers and the Waiters.  It may be convenient to fold the waiter parameters into a new self.waiter_config dict in the Operator's init and just unpack it in the Trigger and Waiter calls, something like:

```
self.waiter_config = {
    cluster_arn=cluster_details["clusterArn"],
    waiter_delay=self.waiter_delay,
    waiter_max_attempts=self.waiter_max_attempts,
    aws_conn_id=self.aws_conn_id,
    region=self.region,
}
```

then all of the waiters and triggers can reuse that cleanly like:

```trigger=ClusterActiveTrigger(**self.waiter_config)```

Reply:
oorrrrrrr maybe that's another use for a BaseAwsOperator class... it could have a helper method that generates a `self.trigger_config` and/or a `self.waiter_config` in the constructor...

Reply:
I'll buy that.  We should keep a list somewhere of all these little "would be nice in a BaseAwsOperator" things... I can never think of one big convincing reason to put in the effort, but there have been a few smaller conveniences here and there that I don't remember.